calico/tests/conftest.py

```diff
@@ -42,20 +38,29 @@ def setup_calico():
         default --patch '{"spec":{"prometheusMetricsEnabled": true}}'"""
     )
 
+    time.sleep(10)
+
 
 @pytest.fixture(scope='session')
 def dd_environment():
 
     with kind_run(conditions=[setup_calico], kind_config=path.join(HERE, 'kind-calico.yaml')) as kubeconfig:
-        with ExitStack() as stack:
-            calico_host, calico_port = stack.enter_context(
-                port_forward(kubeconfig, 'kube-system', 9091, 'service', 'felix-metrics-svc')
-            )
+        with port_forward(kubeconfig, 'kube-system', 9091, 'service', 'felix-metrics-svc') as (
```

Comment:
Suggestion: We could collapse this into a semantically equivalent single `with` statement and reduce a level of nesting with the `with A() as a, B() as b:` syntax.